README.md

```diff
@@ -284,165 +284,77 @@ Library specific options:
-                GETTEXT_BIN_DIR          DIR/gettext/bin
-                GETTEXT_INCLUDE_DIR      DIR/gettext/include
-                GETTEXT_LIBRARIES        DIR/gettext/lib/intl.lib
-                GETTEXT_MSGFMT           DIR/gettext/bin/msgfmt
-
-	* If CMake complains it couldn't find SQLITE3, choose "Advanced" box on the
-	  right top corner, then specify the location of SQLITE3_INCLUDE_DIR and
-	  SQLITE3_LIBRARY manually.
-	* If you want to build 64-bit minetest, you will need to build 64-bit version
-	  of irrlicht engine manually, as only 32-bit pre-built library is provided.
-	* Hit "Configure"
-	* Hit "Configure" once again 8)
-	* If something is still coloured red, you have a problem.
-	* Hit "Generate"
-	If using MSVC:
-		* Open the generated minetest.sln
-		* The project defaults to the "Debug" configuration. Make very sure to
-		  select "Release", unless you want to debug some stuff (it's slower
-		  and might not even work at all)
-		* Build the ALL_BUILD project
-		* Build the INSTALL project
-		* You should now have a working game with the executable in
-			DIR/minetest-install/bin/minetest.exe
-		* Additionally you may create a zip package by building the PACKAGE
-		  project.
-	If using MinGW:
-		* Using the command line, browse to the build directory and run 'make'
-		  (or mingw32-make or whatever it happens to be)
-		* You may need to copy some of the downloaded DLLs into bin/, see what
-		  running the produced executable tells you it doesn't have.
-		* You should now have a working game with the executable in
-			DIR/minetest/bin/minetest.exe
-
-### Bat script to build Windows releases of Minetest
-
-This is how we build Windows releases.
-
-    set sourcedir=%CD%
-    set installpath="C:\tmp\minetest_install"
-    set irrlichtpath="C:\tmp\irrlicht-1.7.2"
-
-    set builddir=%sourcedir%\bvc10
-    mkdir %builddir%
-    pushd %builddir%
-    cmake %sourcedir% -G "Visual Studio 10" -DIRRLICHT_SOURCE_DIR=%irrlichtpath% -DRUN_IN_PLACE=TRUE -DCMAKE_INSTALL_PREFIX=%installpath%
-    if %errorlevel% neq 0 goto fail
-    "C:\WINDOWS\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe" ALL_BUILD.vcxproj /p:Configuration=Release
-    if %errorlevel% neq 0 goto fail
-    "C:\WINDOWS\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe" INSTALL.vcxproj /p:Configuration=Release
-    if %errorlevel% neq 0 goto fail
-    "C:\WINDOWS\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe" PACKAGE.vcxproj /p:Configuration=Release
-    if %errorlevel% neq 0 goto fail
-    popd
-    echo Finished.
-    exit /b 0
-
-    :fail
-    popd
-    echo Failed.
-    exit /b 1
+### Requirements
+
+* Visual Studio 2015 or newer https://visualstudio.microsoft.com
+* CMake https://cmake.org/download/
+* vcpkg https://github.com/Microsoft/vcpkg
+* git https://git-scm.com/downloads
+
+### Compiling and Installing the dependencies
+
+Minetest relays on varios libaries. some are optional and only provide different database backends, and some are mandatory.
```

Comment:
This sentence is _somewhat_ redundant, as the `#### a) Using vcpkg to install dependencies` section already lists the optional ones.

Reply:
```suggestion
Minetest relays on various third party libraries. 
```
Thanks for your review. 
What do you think about my suggestions?
I does not want to omit the paragraph completely as there is missing a introduction then.

Reply:
But is there a need for this section at all? It's quite obvious MT depends on various third-party libs. :)

Reply:
You are right.